examples/amp-date-countdown.amp.html

```diff
@@ -0,0 +1,143 @@
+<!--
+  Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```

Comment:
nit: 2018. Make sure to update this in all new files, but not in files that existed already